src/App.tsx

```diff
@@ -1,18 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
```

Comment:
What do 1 and 42 mean? Move it to variables.
```suggestion
const items = getNumbers(1, 42) .map(n => `Item ${n}`);
```